fxmanifest.lua

```diff
@@ -10,7 +10,7 @@ ox_lib 'locale'
 shared_scripts {
     '@ox_lib/init.lua',
     '@qbx_core/modules/lib.lua',
-    'config.lua'
+    'config/shared.lua'
```

Comment:
This should be in the files section not as a shared script as we don't need to execute it